common/libs/wallet/deterministic/hardware.ts

```diff
@@ -0,0 +1,17 @@
+import { DeterministicWallet } from './deterministic';
+
+export interface ChainCodeResponse {
+  chainCode: string;
+  publicKey: string;
+}
+
+export class HardwareWallet extends DeterministicWallet {
+  // @ts-ignore
+  public static getChainCode(dpath: string): Promise<ChainCodeResponse> {
+    throw new Error(`getChainCode is not implemented in ${this.constructor.name}`);
```

Comment:
Do we want to have these i18n'd?

Reply:
Nah, this is meant for developers. I wanted to make this abstract, but TypeScript doesn't like `static abstract` methods, so I wasn't sure what to do.